### DISCUSSION TITLE
ProPASS Criteria

### DISCUSSION THREAD

**Discussion by AbdulAlaqil** [Ergo-Tools/ActiPASS#61] [General]
Dear Pasan 

Based on ProPASS criteria, I would like to ask what is the minimum time of waking hours and sleep hours. For example, if a person is awake for 18 hours and sleeps for 2hr will that be considered a valid day? 
https://github.com/Ergo-Tools/ActiPASS/wiki/ProPASS-daily-validity-criteria

Thank you, 

Abdul

**Comment by pasan-h** [comment]
> Based on ProPASS criteria, I would like to ask what is the minimum time of waking hours and sleep hours. For example, if a person is awake for 18 hours and sleeps for 2hr will that be considered a valid day? https://github.com/Ergo-Tools/ActiPASS/wiki/ProPASS-daily-validity-criteria

Yes, according to ProPASS daily validity criteria, as you can see, the example you mention is still a valid day. But we recommend using the [ActiPASS_Daily_LongFormat_MasterFile.csv](https://github.com/Ergo-Tools/ActiPASS/wiki/ActiPASS-project-folder-structure#6-actipass_daily_longformat_masterfilecsv) and define your own daily validity criteria if necessary.
